### ISSUE TITLE
We Need Additional Spanish Language support can you please add the spanish langauge our customers need it we already in PROD

### ISSUE THREAD

**Issue by channaveer1** [networknt/json-schema-validator#1025]
@networknt/json-schema-validator Please do the favour atmost we need i18 spanish language.

**Comment by channaveer1** [comment]
@stevehu i asked for help because the build pipeline is failing for java8. the conversion i did correctly still the pipeline is failing can someone help to resolve that

**Comment by channaveer1** [comment]
@stevehu please check this below
https://github.com/networknt/json-schema-validator/pull/1021

**Comment by stevehu** [comment]
It failed due to the following error. I am not familiar with i18n but I am guessing it is an encoding issue.

```
[ERROR]   LocaleTest.encoding:176 expected: <$: debe tener como máximo 5 caracteres> but was: <$: debe tener como mÃ¡ximo 5 caracteres>

```

**Comment by channaveer1** [comment]
@stevehu yes i asked for encoding issue help only it failing because of encoding locally its passing i have tested the encoding if someone could help me to resolve this issue would be a favour. thank you

**Comment by channaveer1** [comment]
@stevehu any update please

**Comment by channaveer1** [comment]
@stevehu Please resolve this issue as early as possible i would be great help

**Comment by stevehu** [comment]
Close it as there is no actions.
